aws/lambda/oss_ci_job_queue_time/Makefile

```diff
@@ -0,0 +1,19 @@
+all: run-local
+
+clean:
+	rm -rf deployment
+	rm -rf venv
+	rm -rf deployment.zip
+
+venv/bin/python:
+	virtualenv venv
+	venv/bin/pip install -r requirements.txt
+
+deployment.zip:
```

Comment:
You want to include this deployment.zip in the release of test-infra versions. 

https://github.com/pytorch/test-infra/blob/main/.github/workflows/lambda-do-release-runners.yml#L83